src/pattern-library/components/patterns/PanelPatterns.js

```diff
@@ -1,39 +1,46 @@
-import {
-  PatternExample,
-  PatternExamples,
-  PatternPage,
-  Pattern,
-} from '../PatternPage';
+import Library from '../Library';
 
 import { IconButton, LabeledButton } from '../../../';
 
 export default function OrganismPatterns() {
   return (
-    <PatternPage title="Panels">
+    <Library.Page title="Panels">
       <p>
         The <code>panel</code> pattern is used by several components:{' '}
         <code>Panel</code>, <code>Dialog</code>, <code>Modal</code>, e.g.
```

Comment:
An idea for a future iteration: maybe `<code>Panel</code>, <code>Dialog</code>, <code>Modal</code>` should replaced by links?

Reply:
Indeed: https://github.com/hypothesis/frontend-shared/issues/103 -- already tracked!